src/Value/Date.php

```diff
@@ -47,7 +50,7 @@ public function format($format = 'Y-m-d')
     /**
      * Compare this Date with another given Date object.
      *
-     * @param ValueInterface|Date $object
+     * @param \DigipolisGent\Value\ValueInterface|\StadGent\Services\OpeningHours\Value\Date $object
```

Comment:
Is Date ook niet van het type ValueInterface?

Reply:
Yes, but the reason I use the `\DigipolisGent\Value\ValueInterface|\StadGent\Services\OpeningHours\Value\Date` here is to make autocomplete work ;-)